test/linreg/genData.py

```diff
@@ -0,0 +1,37 @@
+# Samuel Havron
+# https://www.github.com/samuelhavron
+# MIT License
```

Comment:
Yes, you're quite welcome to change that to be flush with the rest of the repo's BSD license!